packages/client/tests/functional/query-validation/tests.ts

```diff
@@ -0,0 +1,394 @@
+                      ~~~~~~
+                      include: {}
+                      ~~~~~~~
+                    })
+
+            Please either use \`include\` or \`select\`, but not both at the same time.
+        `)
+  })
+
+  test('include used on scalar field', async () => {
+    const result = prisma.user.findMany({
+      // @ts-expect-error
+      include: { id: true },
+    })
+
+    await expect(result).rejects.toMatchPrismaErrorInlineSnapshot(`
+
+            Invalid \`prisma.user.findMany()\` invocation in
+            /client/tests/functional/query-validation/tests.ts:0:0
+
+              XX })
+              XX 
+              XX test('include used on scalar field', async () => {
+            → XX   const result = prisma.user.findMany({
+                     include: {
+                       id: true
+                       ~~
+                     }
+                   })
+
+            Invalid scalar field \`id\` for include statement on model User. Available options are listed in green.
+            Note, that include statements only accept relation fields.
+        `)
+  })
+
+  test('unknown selection field', async () => {
+    const result = prisma.user.findMany({
+      select: {
+        // @ts-expect-error
+        notThere: true,
+      },
+    })
+
+    await expect(result).rejects.toMatchPrismaErrorInlineSnapshot(`
+
+            Invalid \`prisma.user.findMany()\` invocation in
+            /client/tests/functional/query-validation/tests.ts:0:0
+
+              XX })
+              XX 
+              XX test('unknown selection field', async () => {
+            → XX   const result = prisma.user.findMany({
+                     select: {
+                       notThere: true,
+                       ~~~~~~~~
+                   ?   id?: true,
+                   ?   email?: true,
+                   ?   name?: true,
+                   ?   createdAt?: true,
+                   ?   published?: true
+                     }
+                   })
+
+            Unknown field \`notThere\` for select statement on model User. Available options are listed in green.
+        `)
+  })
+
+  test('empty selection', async () => {
+    const result = prisma.user.findMany({
+      select: {},
+    })
+
+    await expect(result).rejects.toMatchPrismaErrorInlineSnapshot(`
+
+            Invalid \`prisma.user.findMany()\` invocation in
+            /client/tests/functional/query-validation/tests.ts:0:0
+
+              XX })
+              XX 
+              XX test('empty selection', async () => {
+            → XX   const result = prisma.user.findMany({
+                     select: {
+                   ?   id?: true,
+                   ?   email?: true,
+                   ?   name?: true,
+                   ?   createdAt?: true,
+                   ?   published?: true
+                     }
+                   })
+
+            The \`select\` statement for type User must not be empty. Available options are listed in green.
```

Comment:
It would be nice to have backticks here (and also any other possibly misisng backtick).